apps/website/sidebars.js

```diff
@@ -5,46 +5,56 @@ const babel = require('@babel/core');
 function generateTryItOutSidebar() {
   const playgroundTemplatePath = path.join(__dirname, '/src/components/Playground/code/templates');
   const templateFiles = fs.readdirSync(playgroundTemplatePath);
-  return templateFiles
-    .map(templateFile => {
-      const id = path.parse(templateFile).name;
-      const templatePath = path.join(playgroundTemplatePath, templateFile);
-      const code = fs.readFileSync(templatePath, { encoding: 'utf-8' });
-      const res = babel.parseSync(code, { filename: templateFile });
-      const meta = { name: id };
-      babel.traverse(res, {
-        VariableDeclarator(path) {
-          const idPath = path.get('id');
-          const initPath = path.get('init');
-
-          if (idPath.isIdentifier({ name: 'meta' }) && initPath.isObjectExpression) {
-            const properties = initPath.get('properties');
-            if (Array.isArray(properties)) {
-              properties.forEach(propertyPath => {
-                if (propertyPath.isObjectProperty()) {
-                  const keyPath = propertyPath.get('key');
-                  const valuePath = propertyPath.get('value');
-
-                  if (keyPath.isIdentifier() && (valuePath.isStringLiteral() || valuePath.isBooleanLiteral())) {
-                    meta[keyPath.node.name] = valuePath.node.value;
-                  }
+  const sidebarItems = [];
+
+  templateFiles.forEach(templateFile => {
```

Comment:
Content of the loop is more or less the same, difference now is that `position` is handled on items.

Reply:
Makes sense, done